v2/kafka-to-pubsub/src/main/java/com/google/cloud/teleport/v2/kafka/consumer/SslConsumerFactoryFn.java

```diff
@@ -120,4 +130,10 @@ private static void copyGcsFileToLocal(String gcsFilePath, String outputFilePath
     }
     throw new IOException("Failed to write file");
   }
+
+  private static void validateCertificate(String storePath, String password, String keystoreType)
```

Comment:
Could you please add a docstring to describe how exactly the validation is performed? For now, it doesn't seem so trivial, I think it would be great if this could be explained